.github/workflows/periodic-rebuild.yaml

```diff
@@ -0,0 +1,37 @@
+name: Scheduled Re-Build Images
+
+on:
+  schedule:
+  # every night at 2am
+  - cron: "0 2 * * *"
+
+jobs:
+  re-build-images:
+    strategy:
+      fail-fast: false
+      matrix:
+        repo: ["mlrun","ui"]
+        branch: ["development","1.3.x"]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Re-Build MLRun Image
+        if: matrix.repo == 'mlrun'
+        uses: convictional/trigger-workflow-and-wait@v1.6.5
+        with:
+            owner: mlrun
+            repo: mlrun
+            github_token: ${{ secrets.RELEASE_GITHUB_ACCESS_TOKEN }}
+            workflow_file_name: build.yaml
+            ref: ${{ matrix.branch }}
+            wait_interval: 60
+            client_payload: '{"skip_images": "models-gpu,tests", "build_from_cache": "false"}'
```

Comment:
I think you should also skip models and base because we made effort to align models and models-gpu (and base is a base image for both)